apps/ledger-live-desktop/src/renderer/modals/UpdateFirmwareModal/steps/01-step-prepare.tsx

```diff
@@ -113,57 +111,31 @@ const Body = ({
             </>
           </Flex>
         )}
-
         <Title>{t("manager.modal.confirmIdentifierText")}</Title>
       </>
     );
   } else {
-    const isStepTransfer = step !== "transfer";
-
+    const onTransferStep = step === "transfer";
```

Comment:
i guess `isStepTransfer` was more explicit naming for this value

Reply:
It's a prepositional phrase, `onWhichStep` or `isOnWhichStep`. 😄